resources/views/pages/achievement/[achievement]/comments.blade.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+use App\Models\Achievement;
+use Illuminate\View\View;
+
+use function Laravel\Folio\{middleware, name, render};
+
+middleware(['auth', 'can:viewAny,' . Achievement::class]);
```

Comment:
```suggestion
middleware(['can:viewAny,' . Achievement::class]);
```

If I'm not mistaken, this is currently public-facing. We'll probably need a `pageDescription` too.

Reply:
The description defaults to the page title, but I've added something more descriptive, and added the icon too.